src/layouts/Main/Sidebar.js

```diff
@@ -1,8 +1,13 @@
 import m from "mithril";
 import stream from "mithril/stream";
+import ConchVersions from "../../models/ConchVersions";
```

Comment:
I think I've managed to set things up so we can use paths from "src/" so you can use `models/ConchVersions` here rather than needing the `../../`.